src/commands/createWebApp/setAppWizardContextDefault.ts

```diff
@@ -40,19 +36,11 @@ export async function setAppWizardContextDefault(wizardContext: IAppServiceWizar
                 LinuxRuntimes.tomcat,
                 LinuxRuntimes.wildfly
             ];
-
-            // considering high resource requirement for Java applications, a higher plan sku is set here
-            wizardContext.newPlanSku = { name: 'P1v2', tier: 'PremiumV2', size: 'P1v2', family: 'P', capacity: 1 };
```

Comment:
Agreed, but I had to move it because that logic is determining recommended runtimes based on projects.  But if you create a Java app with no project opened, you don't get the defaults we recommend.  

We can determine the new runtime by this point so we should leverage that information.